agml/synthetic/manual.py

```diff
@@ -186,8 +192,13 @@ def generate_manual_data(code = None,
 
     # Run the generation.
     executable = os.path.join(project_path, 'build', project_name)
+    os.chmod(executable, stat.S_IRUSR | stat.S_IWUSR | stat.S_IXUSR | stat.S_IRGRP | stat.S_IXGRP |
+             stat.S_IROTH | stat.S_IXOTH) # add necessary permissions to executable
+    env = os.environ.copy()
+    if os.path.exists('/usr/lib/wsl/lib'): # if using WSL, this is mandatory to use radiation plugin
+        env['LD_LIBRARY_PATH'] += ':/usr/lib/wsl/lib'
```

Comment:
It is mandatory to add `/usr/lib/wsl/lib` to `LD_LIBRARY_PATH` when running any project that uses the radiation plugin through WSL due to the OptiX driver requirements.